spec.bs

```diff
@@ -1134,6 +1040,163 @@ To <dfn>convert a modifier to a string</dfn> given a [=part/modifier=] |modifier
   1. Return the empty string.
 </div>
 
+<h2 id=canon>Canonicalization</h2>
+
+<h3 id=canon-encoding-callbacks>Encoding Callbacks</h3>
+
+<div algorithm>
```

Comment:
I assume these will all have more than one call site eventually? Right now it seems like they could just be inlined into the "proccess for init" counterparts.

Reply:
Yes, they are used in the next pull request.